respa_admin/static_src/js/resourceFormLanguage.js

```diff
@@ -0,0 +1,98 @@
+/*
+* Toggle language based on entered language for all fields in the
+* whole form.
+*
+* @param {String} language  Language provided by the DOM value.
+* */
+export function toggleLanguage(language) {
+  if (language === getCurrentLanguage()) { return; }
```

Comment:
Why current language cannot be toggled?

Reply:
It is to prevent the user from hiding all of the translated fields from the UI.

Reply:
IMHO it's more harmful than useful. It makes the UI feel broken.